### PR TITLE
hypershift-qe: delete the hypershift part of 4.11

### PR DESCRIPTION
**Pull request by LiangquanLi930**
Now we don't need hypershift tests in 4.11, so we removed them

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11.yaml [hunk 1]
```diff
@@ -122,7 +122,6 @@ tests:
         curl -s -k -L ${MIRROR2URL}/x86_64/clients/operator-sdk/${optsdkver}/operator-sdk-linux-x86_64.tar.gz -o opt-sdk.tar.gz
         tar -C ${HOME} -xzvf opt-sdk.tar.gz && rm -fr opt-sdk.tar.gz
 
-        oc image extract quay.io/openshifttest/hypershift-client:latest --file=/hypershift && mv hypershift "${HOME}" && chmod 755 "${HOME}"/hypershift
```
Comment (heliubj18) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11.yaml:125]: better to keep it

**Comment by LiangquanLi930** [comment]
@kuiwang02 @liangxia could you help to review this PR? thanks

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11.yaml (hunk 1 above)
Comment (heliubj18) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11.yaml]: @kuiwang02 FYI.

**Pull request merged**